ui/components/component-library/avatar-favicon/avatar-favicon.js

```diff
@@ -48,23 +53,23 @@ export const AvatarFavicon = ({
 
 AvatarFavicon.propTypes = {
   /**
-   * The imageSource accepts the string of the image to be rendered
+   * The src accepts the string of the image to be rendered
    */
-  imageSource: PropTypes.string,
+  src: PropTypes.string,
   /**
    * The alt text for the favicon avatar to be rendered
    */
-  imgAlt: PropTypes.string,
+  name: PropTypes.string,
```

Comment:
Let's make this a required prop to promote good accessibility. Could we also add a section in the docs for Name. That way the alt tags will be a bit more descriptive 

<img width="1436" alt="Screenshot 2022-12-20 at 5 31 43 PM" src="https://user-images.githubusercontent.com/8112138/208799640-2d74ab09-812d-4721-976d-3be6e9be8229.png">